### DISCUSSION TITLE
Nested AD groups

### DISCUSSION THREAD

**Discussion by jonra82** [dexidp/dex#2515] [Q&A]
Hi!

Can Dex fetch a user's nested groups via ldap? From Active Directory.

**Comment by nabokihms** [comment]
Hello, @jonra82. As far as I'm concerned, you can use particular values in LDAP queries to fetch nested groups - [LDAP_MATCHING_RULE_IN_CHAIN](https://docs.microsoft.com/ru-ru/windows/win32/adsi/search-filter-syntax?redirectedfrom=MSDN).
